xlators/protocol/server/src/server.c

```diff
@@ -699,7 +695,7 @@ server_graph_janitor_threads(void *data)
 out:
     free(arg->victim_name);
     free(arg);
-    if (!totchildcount && destroy_ctx) {
+    if (!top->children && destroy_ctx) {
```

Comment:
even !top->children is not required here...destroy_ctx would be true if top->children == 0

Reply:
Ahh, right. I Will update that.